converter/internal/prometheusconvert/prometheusconvert.go

```diff
@@ -52,50 +52,41 @@ func Convert(in []byte) ([]byte, diag.Diagnostics) {
 	return prettyByte, diags
 }
 
-type prometheusBlocks struct {
-	discoveryBlocks             []*builder.Block
-	discoveryRelabelBlocks      []*builder.Block
-	prometheusScrapeBlocks      []*builder.Block
-	prometheusRelabelBlocks     []*builder.Block
-	prometheusRemoteWriteBlocks []*builder.Block
-}
-
-func newPrometheusBlocks() *prometheusBlocks {
-	return &prometheusBlocks{
-		discoveryBlocks:             []*builder.Block{},
-		discoveryRelabelBlocks:      []*builder.Block{},
-		prometheusScrapeBlocks:      []*builder.Block{},
-		prometheusRelabelBlocks:     []*builder.Block{},
-		prometheusRemoteWriteBlocks: []*builder.Block{},
-	}
-}
-
 // AppendAll analyzes the entire prometheus config in memory and transforms it
 // into Flow Arguments. It then appends each argument to the file builder.
 // Exports from other components are correctly referenced to build the Flow
 // pipeline.
```

Comment:
Add some documentation around labelPrefix, its not clear what that does from the functional call? Seems like it was jobname and is now label prefix?

Reply:
I added some details about this